x-pack/lib/monitoring/inputs/metrics.rb

```diff
@@ -187,5 +209,20 @@ def fetch_global_stats
         }
       }
     end
+
+    def extract_cluster_uuids(stats)
+      result = stats.extract_metrics([:stats, :pipelines, :main, :config], :cluster_uuids)
+      if result && !result[:cluster_uuids].empty?
+        @logger.info("Found cluster_uuid from elasticsearch output plugins, ignoring setting monitoring.cluster_uuid")
```

Comment:
What do you think about adding a `debug` entry with the list of discovered `cluster_uuid` entries, with a `warning` entry either if `cluster_uuid` entries are found and there is a `monitoring.cluster_uuid` setting in the `logstash.yml`, or no `cluster_uuid` entries are found either in elasticsearch outputs, or in `logstash.yml`?

Reply:
Done with a3c46475d1528469a6d1643181a49886818fec58